### DISCUSSION TITLE
제페토월드 스크립트 불러내는 법? 제발 도와주세요

### DISCUSSION THREAD

**Discussion by K2Ken** [naverz/zepeto-studio-kor#173] [5. Q&A]
제가 외국애 사는 한국인이여서 변역기를 섰스니 문장이 조금 이상할수도 있읍니다

Unity를 사용하여 Zepeto World를 만들려고 하는데 몇 가지 문제가 있습니다. 저는 Zepeto Studio 가이드의 단계별 자습서를 따르고 있었습니다. 저는 Hello ZepetoScript 페이지의 GameObject에 Zepeto Script 파일을 첨부할 때까지 따라갔지만, 그 이후로는 길을 잃었습니다. 무슨 짓을 해도 ZepetoScriptContext script를 'ZepetoScript Context' GameObject에 표시할 수 없었기 때문입니다.

그 다음에는 내가 무엇을 해도 Visual Studio로 스크립트가 열리지않았습니다, 때문에 C# 스크립트를 편집할수없었습니다. 많은 방법을 시도 해봤지만 C# 스크립트가 Microsoft 영화 및 TV에서 계속 열렸고 Visual Studio로 열리면 .cs 파일로 열리지 않았습니다.

어떻게 해야 할지 매우 혼란스럽습니다.
도와주시겠어요? 

참거 파일:
https://studio.zepeto.me/guides/hello-zepetoscript

I am trying to make a Zepeto World using Unity but I am facing some problems. I was following the step by step tutorial on Zepeto Studio's Guides. I followed along until attaching the Zepeto Script file to the GameObject on the Hello ZepetoScript page , but I got lost after that, because no matter what I did I couldn't get the Zepeto script context to appear on the 'Zepeto script context' GmeObject, although it says that it would automatically create it and link it.
Then after that I didn't know how to get into VIsual Studio to edit the C# script, because no matter what I did I couldn't get in to the propper script in Visual Studio. I tried all of the possibilities I could think of to get into it, because it kept opening in the Microsoft Movies and TV's, and if it did go into Visual Studio, it didn't open as a .cs file.

I am very confused on what to do?
Could you please help 

Link of the page;
https://studio.zepeto.me/guides/hello-zepetoscript

**Comment by moondory77-naverz** [comment]
Hello, K2Ken.
First of all, Thank you for your interest in ZEPETO World development.

1. The ZepetoScriptContext of the part you are confused about,
The 'ZepetoScriptContext' Creation part has been automated within the module through updates,
 Based on the Lastest Package(ZEPETO.World 1.1.1), you can omit it and proceed.
We will update the guide documentation again soon.

2. ZEPETO World development is possible through typescript, not C#.
For this reason, please refer to that thread.

[https://github.com/naverz/zepeto-studio-en/discussions/138#discussioncomment-1868185](https://github.com/naverz/zepeto-studio-en/discussions/138#discussioncomment-1868185)

3. Script editing can be done using any IDE, but for ease of debugging
We recommend using Microsoft Visual Studio code and JetBrains Rider.

[https://studio.zepeto.me/guides/debugging](https://studio.zepeto.me/guides/debugging)

4. Learning through ZEPETO Studio's Guide is good, but overall
In order to understand the context of ZEPETO world contents,
It would be good to refer to the Sample Project developed through the ZEPETO script.

[https://github.com/naverz/zepeto-script-sample](https://github.com/naverz/zepeto-script-sample)

[https://github.com/naverz/zepeto-multiplay-example](https://github.com/naverz/zepeto-multiplay-example)

If you have any other question, please comment.
Thank you.

**Reply by K2Ken** [reply]
Thank you SO MUCH!

The problem was that I didn't have Visual Studio Code downloaded on my computer, so the Typescript script was opening with Microsoft Movies and TV's on default. Now that I have download VS Code and set my default script editor as VS Code everything works as it should. Also thanks for letting me know that getting the 'ZepetoScriptContext' wasn't necessary and i could process without it.

Thank you.
